src/utils/api/apiResponses.ts

```diff
@@ -20,3 +20,13 @@ export const mountSuccessResponse = (bodyObject: object): APIGatewayProxyResult
 		body: JSON.stringify(bodyObject),
 	};
 };
+
+export const unauthorizedResponse: APIGatewayProxyResult = {
+	statusCode: 401,
+	body: JSON.stringify({ message: 'Unauthorized' }),
+};
+
+export const forbiddenResponse: APIGatewayProxyResult = {
+	statusCode: 403,
+	body: JSON.stringify({ message: 'Forbidden' }),
```

Comment:
You can make these messages friendly. The 401 could be "No auth token provided." and the 403 could be "You do not have permission to access this resource."